internal/lsp/server.go

```diff
@@ -747,6 +747,8 @@ func (l *LanguageServer) StartTemplateWorker(ctx context.Context) {
 			newContents, err := l.templateContentsForFile(evt.URI)
 			if err != nil {
 				l.logError(fmt.Errorf("failed to template new file: %w", err))
+
+				continue
```

Comment:
This is not the fix, but when there is an error in templating we should skip the rest.